xsuite/src/data/index.test.ts

```diff
@@ -159,6 +159,7 @@ beforeAll(async () => {
     gasLimit: 10_000_000,
   });
   complexContractState = await contract.getSerializableAccount();
+  complexContractState.codeHash = "123412341234...";
```

Comment:
I was referring to the full code hash that follows the pattern given. But it also seems that whatever can be put there and it won't hurt the test (we put "..." and it worked). So let's put something even more explicit: "some code hash".